### PR TITLE
docs: polish throughout

### PR DESCRIPTION
**Pull request by alexcg1**
- docs(datatypes/text): fix wording

Goals:

- [X] check and update documentation, if required. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#documentation-guidelines)

### CODE REVIEW

**Comment by alexcg1** [comment]
Don't merge this til after v0.20 goes out. Let's get that out the door first

**Review by JohannesMessner — APPROVED** [review]

**Review by samsja — APPROVED** [review]

**Pull request merged**
